packages/plugin-page-view-tracking-browser/src/page-view-tracking.ts

```diff
@@ -22,20 +22,32 @@ export const pageViewTrackingPlugin: CreatePageViewTrackingPlugin = (options: Op
   let localConfig: BrowserConfig;
   const { trackOn, trackHistoryChanges, eventType = defaultPageViewEvent } = options;
 
+  const getDecodeURI = (locationStr: string): string => {
+    let decodedLocationStr = locationStr;
+    try {
+      decodedLocationStr = decodeURI(locationStr);
+    } catch (e) {
+      /* istanbul ignore next */
+      loggerProvider?.log('Malformed URI sequence: ', e);
+    }
+
+    return decodedLocationStr;
+  };
+
   const createPageViewEvent = async (): Promise<Event> => {
+    const locationHref = getDecodeURI(location.href);
```

Comment:
location can be undefined given the current code,  even though I don't get it why as location always exist in a browser environment. 
```suggestion
    const locationHref = getDecodeURI((typeof location !== 'undefined' && location.href) || '');
```